x-pack/plugins/security_solution/public/common/components/callouts/callout.tsx

```diff
@@ -52,8 +55,6 @@ const getDefaultIconType = (type: CallOutType): string => {
       return 'help';
     case 'danger':
       return 'alert';
-    default:
-      return '';
```

Comment:
NOTE: removing the default switch here activates TypeScript's exhaustive match capability to where if another property is added to `CallOutType`, then TypeScript will complain here about the new property not being handled and force us to handle it which looks like is what we want.

Reply:
Can we use `assertUnreachable` in `default` if it gives a more clear error message than this one?

![](https://puu.sh/HeRUU/4d48d1b822.png)

Reply:
![](https://puu.sh/Hf8jS/0cfaeedc02.png)

Yess 👍 Liking this a lot, very useful trick. Thank you for introducing it to our codebase!